internal/host/host.go

```diff
@@ -255,19 +255,19 @@ func (m *Manager) populateDisksEligibility(inventoryString string) (string, erro
 // was set is no longer part of the inventory, the new installation disk is re-evaluated because it is not longer
 // a valid choice.
 func determineDefaultInstallationDisk(previousInstallationDisk string, validDisks []*models.Disk) string {
```

Comment:
We need to think carefully about backwards compatibility concerns. I don't think it's possible to simply change the meaning of this field all of a sudden. It has the potential to break clusters that are discovering during the rollout of these changes.

I think we might need to add a new field here - installation disk ID, that will live along side installation disk path, which we will have to more carefully deprecate...

Reply:
Another option would be to have intelligent migration to the installation_disk_path to reflect the new calculated value.  You are bound to have the same problem with installation_disk_id with discovered hosts. If they were selected before than suddenly the selection disappear with the migration.

Reply:
@slaviered please see https://issues.redhat.com/browse/MGMT-4049